### PR TITLE
Prowjob parser

### PR DESCRIPTION
**Pull request by dekiel**
Created prowjobparser, a helper tool for searching prowjobs according to incude and exclude preset filters.

### CODE REVIEW

File: development/prowjobparser/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# prowjobparser
+
+## Overview
+
+Prowjobparser is a helper tool which parse all prowjobs under provided path, match them against provided labels filters and print matching prowjob names to stdout.
```
Comment (majakurcius) [inline development/prowjobparser/README.md:5]:
```suggestion
The prowjobparser is a helper tool which parses all ProwJobs under the provided path, matches them against the provided label filters, and prints matching ProwJob names to the standard output.
```

File: development/prowjobparser/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Usage
+
+### CLI parameters
+
+The prowjobsparser accepts the following command line parameters:
+
+|Parameter | Shorthand | Description |
+|-----------|-----------|------------|
+| **configpath** | **-c** | Path to prow config yaml file. |
```
Comment (majakurcius) [inline development/prowjobparser/README.md:15]:
```suggestion
| **configpath** | **-c** | Path to the Prow config YAML file. |
```

File: development/prowjobparser/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+| **jobpath** | **-j** | Path to directory containing yaml files with prowjobs. |
```
Comment (majakurcius) [inline development/prowjobparser/README.md:16]:
```suggestion
| **jobpath** | **-j** | Path to the directory containing YAML files with ProwJobs. |
```

File: development/prowjobparser/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+| **includepreset** | **-i** | Preset name which must be added to prowjob. Accept multiple param instances. |
```
Comment (majakurcius) [inline development/prowjobparser/README.md:17]:
```suggestion
| **includepreset** | **-i** | Preset name which must be added to ProwJobs. Accepts multiple-parameter instances. | 
```
Please check if the suggestion reflects the original meaning.

File: development/prowjobparser/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+### CLI parameters
+
+The prowjobsparser accepts the following command line parameters:
+
+|Parameter | Shorthand | Description |
+|-----------|-----------|------------|
+| **configpath** | **-c** | Path to prow config yaml file. |
+| **jobpath** | **-j** | Path to directory containing yaml files with prowjobs. |
+| **includepreset** | **-i** | Preset name which must be added to prowjob. Accept multiple param instances. | 
+| **excludepreset** | **-e** | Preset name which shouldn't be added to prowjob. Accept multiple param instances. |
```
Comment (majakurcius) [inline development/prowjobparser/README.md:18]:
```suggestion
| **excludepreset** | **-e** | Preset name which must not be added to ProwJobs. Accepts multiple-parameter instances. | 
```

File: development/prowjobparser/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+### CLI parameters
+
+The prowjobsparser accepts the following command line parameters:
+
+|Parameter | Shorthand | Description |
+|-----------|-----------|------------|
+| **configpath** | **-c** | Path to the Prow config YAML file. |
+| **jobpath** | **-j** | Path to the directory containing YAML files with ProwJobs. |
+| **includepreset** | **-i** | Preset name which must be added to ProwJobs. Accepts multiple-parameter instances. | 
+| **excludepreset** | **-e** | Preset name which must not be added to ProwJobs. Accepts multiple-parameter instances. |
```
Comment (majakurcius) [inline development/prowjobparser/README.md:18]: Shouldn't we also have a section on how to run it?

Comment (dekiel) [reply development/prowjobparser/README.md:20]: Added example execution. There is no prerequisites, it's quite simple tool.

File: development/prowjobparser/README.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# prowjobparser
+
+## Overview
+
+The prowjobparser is a helper tool which parses all ProwJobs under the provided path, matches them against the provided label filters, and prints matching ProwJob names to the standard output.
+
+## Usage
+
+ `~/go/src/github.com/dekiel/test-infra/development/prowjobparser$ go run main.go -c ../../../../kyma-project/test-infra/prow/config.yaml -j ../../../../kyma-project/test-infra/prow/jobs -i preset-sa-gke-kyma-integration -e preset-sa-kyma-artifacts`
```
Comment (majakurcius) [inline development/prowjobparser/README.md:9]:
``````suggestion
To use the prowjobparser, run it with appropriate parameters. See the example:

```bash
go run main.go \
  -c ../../../../kyma-project/test-infra/prow/config.yaml \
  -j ../../../../kyma-project/test-infra/prow/jobs \
  -i preset-sa-gke-kyma-integration \
  -e preset-sa-kyma-artifacts
```
``````

**Pull request merged**